source/_integrations/group.markdown

```diff
@@ -405,7 +405,9 @@ When member entities all have a single `on` and `off` state, the group state wil
 | person         | home     | not_home |
 | media_player   | ok       | problem  |
 
-When a group contains entities from domains that have multiple `on` states or only use `on` and `off`, the group state will be `on` or `off`.
+When a group contains entities from domains that have multiple `on` states, and more than one of these `on` states are actively used, or only use `on` and `off`, the group state will be `on` or `off`.
```

Comment:
```suggestion
When a group contains entities from domains that have multiple `on` states, the **group** state will be `on` or `off` if one of the following is true:
- More than one of these `on` states are actively used.
- The entities only use `on` and `off`.
```
This sentence is a bit complex. Trying to break this up into a list in an attempt to make it easier to read. If it is incorrect or you don't like it for some other reason, feel free to ignore or change.

Reply:
Thnx